tests/expectations/core/test_expect_column_values_to_be_in_set.py

```diff
@@ -14,9 +14,6 @@
 # <snippet name="tests/expectations/core/test_expect_column_values_to_be_in_set.py ExpectColumnValuesToBeTwoLetterCountryCode_class_def">
 class ExpectColumnValuesToBeTwoLetterCountryCode(ExpectColumnValuesToBeInSet):
     value_set: List[str] = ["FR", "DE", "CH", "ES", "IT", "BE", "NL", "PL"]
-    default_kwarg_values = {
-        "value_set": ["FR", "DE", "CH", "ES", "IT", "BE", "NL", "PL"],
-    }
```

Comment:
I believe I fixed one other test like this - we should see it in CI but I'll take a look to try and find it

Reply:
Was it `ExpectColumnValuesAsStringToBePositiveInteger`? I just removed it from there too.